pkg/testsuite/mock/wallet_transactions.go

```diff
@@ -390,7 +389,7 @@ func (w *Wallet) SendFundsFromAccount(transactionName string, accountOutputName
 		},
 		Features: iotago.BasicOutputFeatures{},
 	}}
-	signedTransaction := w.createSignedTransactionWithOptions(
+	signedTransaction := w.CreateSignedTransactionWithOptions(
```

Comment:
My reason for keeping this method private was to declutter the tests themselves a bit by creating methods methods that create specific types of transactions as the public methods used in the tests.